packages/block-editor/src/components/block-mobile-toolbar/block-actions-menu.native.js

```diff
@@ -426,12 +421,12 @@ export default compose(
 					);
 				},
 				duplicateBlock() {
-					return duplicateBlocks( clientIds );
+					return duplicateBlocks( [ clientId ] );
```

Comment:
FWIW, this should also work with a single value since `getBlocksByClientId()` is used inside of `duplicateBlocks`, and that performs a cast to an array.

However, in favor of better use of the API, I'm happy to keep the specific array here, given that the function is in plural.